config/install_smdebug.sh

```diff
@@ -2,17 +2,36 @@
 set -ex
 set -o pipefail
 
+CORE_REPO="https://github.com/awslabs/sagemaker-debugger.git"
+RULES_REPO="https://github.com/awslabs/sagemaker-debugger-rules.git"
+SMDEBUG_S3_BINARY="s3://smdebug-nightly-binaries/$(date +%F)/"
+
+echo "Cloning sagemaker-debugger and sagemaker-debugger-rules repository."
+cd
+git clone "$CORE_REPO"
+git clone "$RULES_REPO"
+
+export CODEBUILD_SRC_DIR=`pwd`/sagemaker-debugger
+export CODEBUILD_SRC_DIR_RULES=`pwd`/sagemaker-debugger-rules
+
+echo "sagemaker-debugger repository cloned in the path $CODEBUILD_SRC_DIR"
+echo "sagemaker-debugger-rules repository cloned in the path $CODEBUILD_SRC_DIR_RULES"
 
 cd $CODEBUILD_SRC_DIR
-# you can provide bip binary as s3 path in the build environment
```

Comment:
We want both ways - 
1) Running in pipeline, which would work on binary
2) Just run build on individual branch and commit, which would require existing way.